### DISCUSSION TITLE
blog/2022-in-review

### DISCUSSION THREAD

**Discussion by giscus** [Moshyfawn/moshyfawn#27] [Announcements]
# blog/2022-in-review

Me reflecting on my experience of the  2022 - the toughest year for me yet

https://moshyfawn.dev/blog/2022-in-review

**Comment by TkDodo** [comment]
Dominik The Writer - I might have to steal that one :)

Thank you for all your work and our interactions last year - and to an even better 2023 🥂

**Reply by Moshyfawn** [reply]
> Dominik The Writer - I might have to steal that one :)

Please, do! That's how I refer to you in my head anyway lol

I'm thankful for our interactions! To an even better 2023 🥂

**Comment by ardeora** [comment]
Really beautifully written ❤️ Can’t wait to read all your blogs! Happy New Year 😄

**Reply by Moshyfawn** [reply]
I appreciate the kind words, Aryan; it means a lot!

Already sketching up a few blog posts. I hope they'll be of use :D

Happy New Year, man!

**Comment by bluebill1049** [comment]
Thank you very much for your work and support over the year! I felt lucky to have you working on the project together. I wish you the best for the 2023 new year. I am working on [BEEKAI](https://www.beekai.com) (going live this year) and hopefully will bring financially sustainable to the project. We all know how much time and effort maintainers have contributed. Having solid funding to back up your contribution and the project will push us further into better form management.

Cheers
BIll

**Reply by Moshyfawn** [reply]
While I couldn't dedicate as much time to maintain RHF as I wanted to, I'm glad I could help 🖤 I really enjoyed working together with you on some of the RHF features of the past year!

You're doing an incredible thing with BEEKAI and I'm positive it'll succeed! OSS supprot and funding is such an important part software development.

Here's to better form management ✨

**Comment by jsun969** [comment]
Cool blog and awesome article! Nice to meet you in 2022! Happy New Year 🎉

**Reply by Moshyfawn** [reply]
Thank you, Justin!

The blog is nothing special as I haven't dedicated nearly enough time to it's design, but I hope it serves it's purpose for now and is OK to look at for the time being :P

Right back at you and I'm glad to work along-side you 🖤 Happy New Year, Justin 🎄
